src/ui/locales/it.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Attiva/disattiva attribuzione",
+    "AttributionControl.MapFeedback": "Feedback sulla mappa",
+    "FullscreenControl.Enter": "Entra a schermo intero",
+    "FullscreenControl.Exit": "Esci dalla modalità schermo intero",
+    "GeolocateControl.FindMyLocation": "Trova la mia posizione",
+    "GeolocateControl.LocationNotAvailable": "Posizione non disponibile",
+    "LogoControl.Title": "Logo della scatola delle mappe",
+    "Map.Title": "Carta geografica",
+    "NavigationControl.ResetBearing": "Ripristina la direzione verso nord",
+    "NavigationControl.ZoomIn": "Ingrandire",
+    "NavigationControl.ZoomOut": "Rimpicciolisci",
+    "ScaleControl.Feet": "piedi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "km",
+    "ScaleControl.Miles": "me",
+    "ScaleControl.NauticalMiles": "nm",
+    "ScrollZoomBlocker.CtrlMessage": "Usa ctrl + scorri per ingrandire la mappa",
```

Comment:
Not sure what keyboard/mosue control is "Scroll" referring to?

Reply:
This is a text on map blocker alert when cooperative gestures are enabled, see https://docs.mapbox.com/mapbox-gl-js/example/cooperative-gestures/

![Screenshot 2022-04-12 at 13 08 03](https://user-images.githubusercontent.com/533564/162959994-784aadce-b2ae-4f93-b555-3c45d1d9bc1c.png)

Reply:
I checked [apple support in Italian for multi touch](https://support.apple.com/it-it/HT204895), `scorri` should be correct.

Reply:
But also in general it is a good translation 👌

Reply:
@ae-mo, could you please approve this PR if everything looks good?